### PR TITLE
add new blogs

### PR DESCRIPTION
**Pull request by hacktivist123**
This PR updates the blog with new external blog posts

### CODE REVIEW

File: src/posts/2023-11-06-cilium-certified-associate/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-11-06T12:33:00.000Z'
+title: 'COMING SOON! Cilium Certified Associate (CCA)'
+ogImage: ogimage.png
+ogSummary: 'The Cloud Native Computer Foundation and Linux Foundation Training and Certification announce the new Cilium Certified Associate (CCA) certification designed for platform and cloud engineers'
```
Comment (xmulligan) [inline src/posts/2023-11-06-cilium-certified-associate/index.md:5]:
```suggestion
ogSummary: 'The Cloud Native Computing Foundation and Linux Foundation Training and Certification announce the new Cilium Certified Associate (CCA) certification designed for platform and cloud engineers'
```

File: src/posts/2023-11-06-isovalent-cilium-certified-associate/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-11-06T12:33:00.000Z'
+title: 'All About The New CNCF Cilium Certified Associate (CCA) Certification'
+ogImage: ogimage.webp
+ogSummary: 'Everything you need to know about the new CNCF Cilium Certified Associate (CCA) Certification and how to study for it'
+categories:
+  - Technology
```
Comment (xmulligan) [inline src/posts/2023-11-06-isovalent-cilium-certified-associate/index.md:7]:
```suggestion
  - How-To
```

File: src/posts/2023-11-06-cilium-certified-associate/index.md (hunk 1 above)
Comment (hacktivist123) [reply src/posts/2023-11-06-cilium-certified-associate/index.md:5]: Ah, good catch, it must have been my vscode autocomplete feature that caused this.

**Comment by hacktivist123** [comment]
@xmulligan all good now, let's ship

**Review by xmulligan — APPROVED** [review]

**Pull request merged**